shell/platform/fuchsia/flutter/platform_view_unittest.cc

```diff
@@ -2,14 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
This file is mostly style cleanups, and API fixes to comply to the new platform_view interface....its a no-op